### PR TITLE
Add components for UserAccountView and UserAccountUpdateView for Frontend

### PR DESCRIPTION
**Pull request by anthony-rk**
### :scroll: Description

- Describe the general shape of this PR (new feature? refactor? bug fix? one-line change?)
-- Add components for the User Account View

- Describe what changes are being made
-- Created 2 Components, UserAccountView which will show the User's info (First Name, Last Name, Email, Phone, Reset Password) and UserAccountViewUpdate which will also show the User's info but in an Editable way.

- Describe why these changes are being made
-- Changes are being made to allow Users to see their info and make changes if needed.

- List the use cases and edge cases relevant to this PR
-- Any User will be able to use this. 

- Describe how errors will be handled. How will we know if this code breaks in production
-- Need to add input checkers (sanitization) for User account editing. 

- Describe any external libraries/dependencies added or removed in this PR
-- N/A

- Describe any security risks are there regarding this change
-- Need to Sanitize user input.

- Describe how you tested these changes

- Link to relevant external documentation

--------------------------
### :clipboard: Mandatory Checklist

- [x] Example of a checked item (please remove when creating your Pull Request) 

- [ ] Linked to the Github Issues being addressed using the right sidebar :arrow_right:
- [ ] Have you discussed these changes with the project leader(s)?
- [ ] Do all variable and function names communicate what they do?
- [ ] Were all the changes commented and / or documented?
- [ ] Is the PR the right size? (If the PR is too large to review, it might be good to break it up into multiple PRs.)
- [ ] Does all work in progress, temporary, or debugger code have a TODO comment with links to Github issues?
- [ ] *If you changed the user interface, did you add before and after screenshots to below?*

--------------------------
#### After
- First section will show the user data, the second section is the user data in edit mode, will need to switch the view components if the user selects "Update Account"
Screen Shot 2021-06-10 at 12 49 32 PM

--------------------------
### :blue_book: Glossary
- PR = Pull Request

### CODE REVIEW

File: frontend/src/components/UserAccountView/UserAccountUpdateView.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import React from 'react';
+
+const UserAccountUpdateForm = () => {
+  const handleResetPassword = () => {
+    alert('Password Reset Link Sent to {your email}');
+  };
+
+  return (
+    <div className="user-data">
+      <form className="form">
```
Comment (JMStudiosJoe) [inline frontend/src/components/UserAccountView/UserAccountUpdateView.jsx:11]: if not using the form functionality then no need for it lets remove it to simplify please.

**Review by JMStudiosJoe — CHANGES_REQUESTED** [review]
is this still a work in progress?

**Comment by JMStudiosJoe** [comment]
can you update PR please?

**Comment by anthony-rk** [comment]
I replaced the forms with divs for now, let me know if this works or if I need to make any other adjustments.

**Review by JMStudiosJoe — APPROVED** [review]

**Pull request merged**